src/CMakeLists.txt

```diff
@@ -130,3 +130,4 @@ add_subdirectory(tool)
 add_subdirectory(test)
 add_subdirectory(platform)
 add_subdirectory(package)
+add_subdirectory(../samples ${CMAKE_CURRENT_BINARY_DIR}/samples)
```

Comment:
Doh! I totally forgot that samples is a peer to src and doesn't share a root w/ a CMakeLists file in it. Let me circle w/ the other devs and get back to you with a new recommendation.

Reply:
I'm interested what @DefaultRyan, @kennykerr  and @Scottj1s think, but I'm thinking we should *NOT* include samples in the /src/cmakelists.txt file. We shouldn't have a root /samples/cmakelists.txt file either. Just whatever the sample needs to build local to the sample folder. For this sample, that does mean having to set the XLANG_LIBRARY_PATH + compiler settings directly in this file rather than inheriting them from root cmake file. But I think it's better for samples to be isolated. Better to have all those definitions self contained for people attempting to learn from the sample than having to dig thru other cmake files to find them.

Reply:
Agreed.

Reply:
Made meta_reader cmake file self-contained.
```
cmake ./meta_reader -B_build -GNinja && ninja -C _build -v
```